vembrane/__init__.py

```diff
@@ -162,12 +172,18 @@ def filter_vcf(
         env["FORMAT"] = Format(idx, formats)
         env["SAMPLES"] = list(record.samples)
 
-        annotations = dict(record.info).get(ann_key, [""])
-        filtered_annotations = [
-            annotation
-            for annotation in annotations
-            if eval_expression(expression, idx, annotation, annotation_keys, env,)
-        ]
+        if expression.filters_annotations():
+            annotations = dict(record.info).get(ann_key, [""])
+            filtered_annotations = [
+                annotation
+                for annotation in annotations
+                if expression.evaluate(idx, annotation, annotation_keys, env,)
+            ]
+        else:
+            annotations = [""]
```

Comment:
`annotations` gets initialized but is never used.

Reply:
it does get used further down!